sdk/iOS/test/MSSyncTableTests.m

```diff
@@ -1584,6 +1584,142 @@ -(void) testPullWithFetchLimitNotMultipleOfFifty
                   @"Invalid URL: %@", thirdRequest.URL.absoluteString);
 }
 
+-(void) testPullTable_UsePullSettings_NegativePageSize
+{
+    MSPullSettings *pullSettings = [MSPullSettings new];
+    pullSettings.pageSize = -1;
+    
+    NSString *pullRequest = [self getPullTableRequestWithPullSettings:pullSettings usePullSettings:YES];
+
+    XCTAssertEqualObjects(pullRequest, @"https://someUrl/tables/TodoNoVersion?$top=50&__includeDeleted=true&$skip=0&__systemProperties=__deleted", "Incorrect pull request");
+}
+
+-(void) testPullTable_UsePullSettings_ZeroPageSize
+{
+    MSPullSettings *pullSettings = [MSPullSettings new];
+    pullSettings.pageSize = 0;
+    
+    NSString *pullRequest = [self getPullTableRequestWithPullSettings:pullSettings usePullSettings:YES];
+    
+    XCTAssertEqualObjects(pullRequest, @"https://someUrl/tables/TodoNoVersion?$top=50&__includeDeleted=true&$skip=0&__systemProperties=__deleted", "Incorrect pull request");
```

Comment:
you've confirmed in all these tests cases that these sets sort identically on 32 & 64 bit runs?